tests/utils.go

```diff
@@ -5,22 +5,35 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"strings"
 
 	. "github.com/onsi/ginkgo"
 
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+
+	clientset "kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned"
+	"kubevirt.io/containerized-data-importer/pkg/util"
 )
 
 var KubectlPath = ""
 var OcPath = ""
 var CDIInstallNamespace = "kube-system"
 
+const (
+	AlpineUrl         = "http://cdi-http-import-server.kube-system/images/alpine.iso"
```

Comment:
all of our existing tests using tinyCore.iso. I like the idea of including additional images. Should we add alpine.iso to _test/images_?

Reply:
Oh, i missed that tinyCore.iso already exists in our repo.  The http importer service container should be serving the images in test/images/ instead of downloading alpine. 

so no, we don't need alpine here I don't think. the isos already in the src tree should be fine